src/main/antlr4/Manifold.g4

```diff
@@ -97,11 +98,6 @@ expression:
   // | declaration
   ;
 
-statement:
-    expression #ExpressionStatment
-    | 'import' STRING_VALUE #ImportStatement
-    ;
-
 STATEMENT_TERMINATOR: ';';
```

Comment:
This was once called `EXPRESSION_TERMINATOR`. I don't see a need to change it now.